/**:
  # Parameters specific to our F710 remote.
  ros__parameters:
    # Steering
    steering_axis: 2          # Right joystick, x direction
    steering_scale: 0.48
    steering_offset: 0.0
    # Speed
    speed_axis: 1             # Left joystick, y direction
    speed_scale: 2.0
    speed_offset: 0.0
    # Buttons
    enable_button: 5          # Enable / disable
    enable_debounce: 0.2      # Debounce (in seconds)

